### PR TITLE
Closes #63 - XString module

### LINKED ISSUE #63: XString
http://package.elm-lang.org/packages/elm-lang/core/5.1.1/String

import String

### PR DESCRIPTION
**Pull request by Baransu**
Closes: #63

### CODE REVIEW

File: src/ElixirStd/XString.elm [hunk 1]
```diff
@@ -0,0 +1,149 @@
+module XString exposing (..)
+
+import Elmchemy exposing (..)
+
+
+-- isEmpty
+
+
+isEmpty : String -> Bool
```
Comment (wende) [inline src/ElixirStd/XString.elm]: Why so many blank lines?

Comment (Baransu) [inline src/ElixirStd/XString.elm]: `elm-format`

**Comment by Baransu** [comment]
It's not related to this PR but I've used `yarn` instead `npm` so now we have Yarn support for development.

**Comment by wende** [comment]
Blocked until #86 is closed

**Pull request merged**